Frameworks/CoreFoundation/Base.subproj/CFSystemDirectories.c

```diff
@@ -55,7 +55,8 @@ CFSearchPathEnumerationState __CFGetNextSearchPathEnumeration(CFSearchPathEnumer
 #endif
 
 
-#if DEPLOYMENT_TARGET_MACOSX || DEPLOYMENT_TARGET_EMBEDDED || DEPLOYMENT_TARGET_WINDOWS
+// WINOBJC: Disable this function only for WinObjC as we don't provide system intrinsics for it.
```

Comment:
That is, `NSGetNextSearchPathEnumeration`

---
In reply to: [72319135](https://github.com/Microsoft/WinObjC/pull/684#discussion_r72319135) [](ancestors = 72319135,72318824)